README.md

```diff
@@ -1,8 +1,8 @@
-# Companies House Late Filing Penalties Web Service
-The Companies House Web Service for handling payments of LFPs. This application is written using the [Spring Boot](http://projects.spring.io/spring-boot/) Java framework.
+# Companies House Penalties Payment Web Service
+The Companies House Web Service for handling payments of penalties. This application is written using the [Spring Boot](http://projects.spring.io/spring-boot/) Java framework.
 
-- Retrieves Payable LFPs from the LFP-PAY-API service
-- Displays payable LFPs and redirects to the Payments Service to take payment.
+- Retrieves Payable penalties from the PPS-PAY-API service
```

Comment:
Update this to  Penalty-Payment-API

Reply:
Changed from PPS-PAY to Penalty-Payment
And PPS_PAY to Penalty_Payment